topi/python/topi/cuda/reduction.py

```diff
@@ -107,7 +107,10 @@ def traverse_before_reduce(operator):
     def traverse_after_reduce(operator):
         """Internal travserse function"""
         if tag.is_broadcast(operator.tag):
-            raise RuntimeError("Not yet support ewise after reduce")
+            for tensor in operator.input_tensors:
+                if tensor.op not in scheduled_ops:
+                    traverse_after_reduce(tensor.op)
+            _schedule_injective(operator, sch)
```

Comment:
sorry I don't understand this change. What is it for?

Reply:
This code supports fused operators which perform an elemwise op after a reduce operator for cuda. 

For another example, nnvm.symbol.mean cannot compilable for cuda. 
```
import nnvm
import nnvm.symbol as sym

x = sym.Variable("x")
y = sym.mean(x, axis=1)
compute_graph = nnvm.graph.create(y)
shape = (4,3, )
deploy_graph, lib, params = nnvm.compiler.build(
    compute_graph, target="cuda", shape={"x": shape}, dtype="float32")
```
The Error is as following. 

  File "/home/morinaga/.local/lib/python2.7/site-packages/topi-0.5.dev0-py2.7.egg/topi/cuda/reduction.py", line 114, in traverse_after_reduce
    raise RuntimeError("Not yet support ewise after reduce")
RuntimeError: Not yet support ewise after reduce